CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 - Bugfix: Fixed timestamps being incorrect on some messages loaded from the recent-messages service on startup (#1286, #2020)
 - Bugfix: Fixed timestamps missing on channel point redemption messages (#1943)
 - Bugfix: Fixed tooltip didn't show in `EmotePopup` depending on the `Link preview` setting enabled or no (#2008)
+- Bugfix: Fixed "Last message line color" button not respecting search queries (#2073)
```

Comment:
No changelog entry is needed if the fix happened to a feature that was added in the same changelog section, you could if anything edit the original feature changelog entry and add this pr number to it

Reply:
Right, my bad. Didn't realize it was unversioned. Reverted in d188eed9.